### PR TITLE
Add riscv64-gnu to self-test

### PR DESCRIPTION
**Pull request by Hoverbear**
Missed this in #1027

### CODE REVIEW

**Comment by Hoverbear** [comment]
bors try

**Comment by Hoverbear** [comment]
bors try

**Comment by Hoverbear** [comment]
bors try

**Comment by Hoverbear** [comment]
bors try

File: ferrocene/ci/scripts/calculate_parameters.py [hunk 1]
```diff
@@ -42,34 +42,36 @@
...
+X86_64_LINUX_SELF_TEST_TARGETS = X86_64_LINUX_BUILD_HOSTS + AARCH64_LINUX_BUILD_HOSTS + X86_64_LINUX_BUILD_STD_TARGETS_ALL
+AARCH64_LINUX_SELF_TEST_TARGETS = X86_64_LINUX_BUILD_HOSTS + AARCH64_LINUX_BUILD_HOSTS  + GENERIC_BUILD_STD_TARGETS
 
 # Targets only built (and tested!) on Mac
-AARCH64_MAC_BUILD_HOSTS = ["aarch64-apple-darwin", "x86_64-apple-darwin"]
-# We don't currently produce x86_64 Apple host tools, but we will one day
-AARCH64_MAC_BUILD_STD_TARGETS = ["x86_64-apple-darwin"]
-AARCH64_MAC_SELF_TEST_TARGETS = AARCH64_MAC_BUILD_HOSTS + AARCH64_MAC_BUILD_STD_TARGETS + X86_64_LINUX_BUILD_STD_TARGETS
+AARCH64_MAC_BUILD_HOSTS = ["aarch64-apple-darwin"]
+AARCH64_MAC_BUILD_STD_TARGETS = ["x86_64-apple-darwin", "x86_64-apple-darwin"]
```
Comment (tshepang) [inline ferrocene/ci/scripts/calculate_parameters.py:69]: perhaps...
```suggestion
AARCH64_MAC_BUILD_STD_TARGETS = ["aarch64-apple-darwin", "x86_64-apple-darwin"]
```

Comment (Hoverbear) [reply ferrocene/ci/scripts/calculate_parameters.py:69]: Oh, great catch. 😬

**Comment by tshepang** [comment]
please squash, either 1 or 2 commits

**Comment by Hoverbear** [comment]
@tshepang done :)

**Review by tshepang — APPROVED** [review]
bors merge

**Comment by Hoverbear** [comment]
(Noting this will almost certainly conflict with #1033 which is also due for merge, I'll resolve and seek re-approval.)

**Review by amanjeev — APPROVED** [review]

**Comment by Hoverbear** [comment]
bors merge

**Pull request merged**